ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,424 @@
+
+TASK [Copy all examples to ~/ansible-avd-examples]*****************************************************************************************************************************************************
+changed: [localhost]
+
+PLAY RECAP
+****************************************************************************************************************************************************************************************************************************************************************************
+localhost                  : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+After the playbook has run successfully, the following directory structure will be created.
+
+```shell
+ansible-avd-examples/     (directory where playbook was run)
+  ├── l2ls-fabric/
+    ├── documentation/
+    ├── group_vars/
+    ├── images/
+    ├── intended/
+    ├── switch-basic-configurations/
+    ├── ansible.cfg
+    ├── build.yml
+    ├── deploy.yml
+    ├── inventory.yml
+    └── README.md (this document)
+```
+
+???+ info
+
+    If the content of any file in the example is ***modified*** and the playbook is run again, the file ***will not*** be overwritten. However, if any file in the example is ***deleted*** and the playbook is run again, the file will be re-created.
+
+## Design Overview
+
+### Physical L2LS Topology
+
+The drawing below shows the physical topology used in this example. The interface assignment shown here are referenced across the entire example, so keep that in mind if this example must be adapted to a different topology.
+
+![Figure: 1](images/pure_L2LS_topo.svg)
+
+???+ note
+
+    The FW/L3 Device and individual hosts (A-D) in this example are not managed by AVD, but the switch ports connecting to these devices are.
+
+## Basic EOS Switch Configuration
+
+Basic connectivity between the Ansible controller host and the switches must be established before Ansible can be used to deploy configurations. The following should be configured on all switches:
+
+- Switch Hostname
+- IP enabled interface
+- Username and Password defined
+- Management eAPI enabled
+
+???+ info
+
+    When using vEOS/cEOS virtual switches, `Management0` is used. When using actual hardware switches, `Management1` is used. The included basic switch configurations may need to be adjusted for your environment.
+
+Below is the basic configuration file for SPINE1:
+
+```shell
+--8<--
+examples/l2ls-fabric/switch-basic-configurations/SPINE1.cfg
+--8<--
+```
+
+## Ansible Inventory
+
+Now that we understand the physical L2LS topology, we need to create the Ansible inventory that represents this topology. The following is a textual and graphical representation of the Ansible inventory group variables and naming scheme used in this example:
+
+``` text
+- DC1
+  - DC1_FABRIC
+    - DC1_SPINES
+    - DC1_LEAFS
+      - POD1
```

Comment:
POD naming can be confusing since PODs in AVD is a full Leaf-Spine setup.
I suggest to rename to `RACK1`, `RACK2`

Reply:
1. Simplified inventory my putting all leafs into DC1_LEAFS
2. Changed all references of POD1 and POD2 to RACK1 and RACK2.
3. Updated drawings the same